### ISSUE TITLE
[Keyboard Navigation - Make code - More options>Download as File]: 'Pair Now' link present under "Download completed" dialog is not keyboard accessible.

### ISSUE THREAD

**Issue by Chandangithb** [microsoft/pxt-microbit#5505]
### User Experience:
Users who rely on keyboard for navigation will not be able to access the "Pair Now" link if the link is not keyboard accessible. Due to which the user will miss the functionality and face challenges in connecting the devices.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:
1. Launch application URL : https://makecode.microbit.org
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "More Options(...)" button and select it.
6. Navigate through the controls appearing select "Download as File"
7. Observe whether 'Pair Now' link present under "Download completed" dialog is keyboard accessible or not.

### Actual Result:
**Issue:** 'Pair Now' link present under "Download completed" dialog is not keyboard accessible.

**Observation:**

- Issue repro with Keyboard only as keyboard focus does not move to "Pair Now' link. Due to which the user is unable to access it with keyboard. However, it works fine with mouse click. 
- User can access the same 'Pair Now' control with keyboard by invoking the first option "Connect Device" under more options. Steps: (Connect Device>Next>Pair)

### Expected Result:
 The 'Pair Now' link present under "Download completed" dialog should be keyboard accessible. The focus should move to the 'Pair Now' link button.

### MAS Reference:
[MAS2.1.1- Keyboard](https://aka.ms/MAS2.1.1)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
Please refer attached screenshot.

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:
- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.

### Attachments:
![MAS2 1 1-'Pair Now' link present under Download completed dialog is not keyboard accessible (Tabstops)](https://github.com/microsoft/pxt-microbit/assets/112375678/1aac3064-36fa-4c7c-a66c-e6fb75108aa2)

https://github.com/microsoft/pxt-microbit/assets/112375678/0b05ac54-537c-495c-bbea-d10ea691fba7

**Comment by Chandangithb** [comment]
#A11yAuto;#Accessibility;#A11yMAS;#A11ySev3;#MAS2.1.1;#Tabstops;#K4W;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Feb2024;
